rdr_common/code_constants.py

```diff
@@ -74,3 +74,94 @@
 UNSET = 'UNSET'
 UNMAPPED = 'UNMAPPED'
 BASE_VALUES = [UNSET, UNMAPPED, PMI_SKIP_CODE]
+
+ps_full_data_headers = [
+ 'PMI ID',
+ 'Biobank ID',
+ 'Last Name',
+ 'First Name',
+ 'Date of Birth',
+ 'Language',
+ 'Participant Status',
+ 'General Consent Status',
+ 'General Consent Date',
+ 'EHR Consent Status',
+ 'EHR Consent Date',
+ 'CABoR Consent Status',
+ 'CABoR Consent Date',
+ 'Withdrawal Status',
+ 'Withdrawal Date',
+ 'Street Address',
+ 'City',
+ 'State',
+ 'ZIP',
+ 'Email',
+ 'Phone',
+ 'Sex',
+ 'Gender Identity',
+ 'Race/Ethnicity',
+ 'Education',
+ 'Required PPI Surveys Complete',
+ 'Completed Surveys',
+ 'Basics PPI Survey Complete',
+ 'Basics PPI Survey Completion Date',
+ 'Health PPI Survey Complete',
+ 'Health PPI Survey Completion Date',
+ 'Lifestyle PPI Survey Complete',
+ 'Lifestyle PPI Survey Completion Date',
+ 'Hist PPI Survey Complete',
+ 'Hist PPI Survey Completion Date',
+ 'Meds PPI Survey Complete',
+ 'Meds PPI Survey Completion Date',
+ 'Family PPI Survey Complete',
+ 'Family PPI Survey Completion Date',
+ 'Access PPI Survey Complete',
+ 'Access PPI Survey Completion Date',
+ 'Physical Measurements Status',
+ 'Physical Measurements Completion Date',
+ 'Paired Site',
```

Comment:
So awardee isn't needed? (I guess it's unnecessary if we only return one awardee's data at a time?)

Reply:
no, and HPRO doesnt return awardee